opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java

```diff
@@ -133,18 +173,36 @@ public void validateArtifactMap() throws InvalidFormatException {
     // no additional artifacts
   }
 
+  /**
+   * @return Retrieves a {@link SequenceCodec} instance.
+   */
   public SequenceCodec<String> createSequenceCodec() {
 
     if (artifactProvider != null) {
-      String sequeceCodecImplName = artifactProvider.getManifestProperty(
+      String sequenceCodecImplName = artifactProvider.getManifestProperty(
           TokenNameFinderModel.SEQUENCE_CODEC_CLASS_NAME_PARAMETER);
-      return instantiateSequenceCodec(sequeceCodecImplName);
+      try {
+        return instantiateSequenceCodec(sequenceCodecImplName);
+      } catch (InvalidFormatException e) {
+        // TODO Is this compensation valid here? Or: should we escalate towards the caller?
+        System.out.println("Encountered a problem with the sequence codec '"
```

Comment:
Is this a problem that we think the user should be aware of immediately?

At some point I'd like to reduce the project's reliance on `System.out.println` so I don't know if using it to tell the user something is a good way to go about it...? I'd probably rather let the exception be thrown and let the application implementing with OpenNLP handle it.

Reply:
I don't know, if it is natural, that we return a `BioCodec` instance or something else from `init(...)` in case we fail to load the requested codec from the given `ArtifactProvider`.  My expectation - as a user - would be, that I get an exception, which tells me, that my `codec`, which I specified via the manifest property couldn't be loaded via the given artifact loader and I can deal with that in my app.

Regarding the second half of @jzonthemtn comment:

We should replace  `System.out.println` and  `System.err.println` with calls to some logging library any time soon. These calls might be thrown into the nether, if OpenNLP is used within an application server for example.

Reply:
Regarding the logging, the project has always tried to maintain a principle of using the fewest dependencies as possible. The log4j vulnerability a few months ago is a good example why. That can certainly be brought up again for discussion on the mailing list, but a better logging solution that doesn't require a new dependency would be ideal (if it even exists).

Reply:
Yes - logging as a topic might be something to discuss on the dev@ list outlining different approaches / possibilities like using a bridge (like slf4j-api) and let the user add the log impl, rely on JUL (no 3rd party deps) and let users add bridges, etc. - can think of some more :)

Reply:
@jzonthemtn I'll remove the sysout here tmrw. The discussion about logger or not seems a separate issue and/or a discussion for the dev lis, that is: can't be solved with this PR.